chart/values.yaml

```diff
@@ -477,6 +477,8 @@ io_engine:
   priorityClassName: ""
   # -- Runtime class to use. Defaults to cluster standard
   runtimeClassName: ""
+  # -- Number of retries for pstor persistence before the volume target self shutdowns
+  pstorRetries: 300
```

Comment:
Should this be 30 now?

Reply:
I think let's leave as 300 to add a bit of buffer before the volume goes read-only